client/components/ConfigureRunsForExample/index.jsx

```diff
@@ -94,11 +95,14 @@ class ConfigureRunsForExample extends Component {
             return b.at_id - a.at_id;
         });
 
+        let exampleTableTitle = example.name || example.directory;
+        let tableId = `table_name_${nextId()}`;
```

Comment:
`nextId()` accepts a prefix argument, so you can just use it as `key={nextId('table_name_')}`